### DISCUSSION TITLE
Why not making the Team Feature as Optional?

### DISCUSSION THREAD

**Discussion by rashidali99** [shipfastlabs/larasonic-vue#13] [Q&A] [answered]
Why not making the Team Feature as Optional?

Just like laravel jetstream 

```bash
php artisan jetstream:install inertia --teams
```

BTW Thanks for the amazing kit.

**Comment by pushpak1300** [comment, marked as answer]
The things is you can easy do it.https://docs.larasonic.com/features/teams#disable-teams

**Reply by rashidali99** [reply]
Thank you for the amazing Starter Kit! I've reviewed the documentation, which is why I closed the discussion. I'm excited about the great features it offers and look forward to seeing even more in the future. I'm also eager to contribute and add value to the project.

Thanks again, and keep up the great work! 👍
